docs/developer-guide/unit-testing.md

```diff
@@ -148,29 +104,29 @@ class Test_iam_password_policy_uppercase:
     # policy we want to set to False the RequireUppercaseCharacters
     iam_client.update_account_password_policy(RequireUppercaseCharacters=False)
 
-    # We set a mocked audit_info for AWS not to share the same audit state
-    # between checks
-    current_audit_info = self.set_mocked_audit_info()
+    # We set a mocked aws_provider info for AWS not to share the same info
+    # between tests
+    aws_provider = set_mocked_aws_provider([AWS_REGION_US_EAST_1])
```

Comment:
Please, explain where does this come from instead of the audit info.

Reply:
Done

Reply:
Include the path here please.